src/platform/site-wide/sass/modules/_m-responsive-table.scss

```diff
@@ -18,7 +16,7 @@ $color-responsive-table-border: #999;
   }
   .va-table--responsive {
     tr {
-      border-top: 2px solid $color-responsive-table-border;
+      border-top: 2px solid $color-gray-light;
```

Comment:
![image](https://user-images.githubusercontent.com/2008881/119393445-ca8f6700-bc85-11eb-8fde-b84bd6cca26d.png)
![image](https://user-images.githubusercontent.com/2008881/119393453-cebb8480-bc85-11eb-95c2-ae6e67d6d03f.png)


This one isn't _as_ close